README.rst

```diff
@@ -2,17 +2,19 @@
 os-net-config
 =============
 
+A declarative network configuration tool for hosts.
+
 Team and repository tags
 ------------------------
 
-.. image:: https://governance.openstack.org/tc/badges/os-net-config.svg
-    :target: https://governance.openstack.org/tc/reference/tags/index.html
+.. image:: https://governance.openstack.org/tc/badges/openapi.svg
+    :target: https://governance.openstack.org/tc/reference/projects/
 
 Overview
 --------
 
 ``os-net-config`` is a host network configuration tool which supports multiple
-backend configuration providers.
+backend configuration providers (One of: ifcfg, eni, nmstate, iproute).
```

Comment:
This should probably read:

backend configuration providers, one of: ifcfg (network-init-scripts), nmstate (NetworkManager), or eni (basic support for /etc/network/interfaces)

We never implemented iproute.